tests/integration/common/transaction_utils.rs

```diff
@@ -736,15 +800,21 @@ async fn execute_txs(
     os_hints(&block_context, state, internal_txs, execution_infos, deprecated_contract_classes, contract_classes).await
 }
 
-pub async fn execute_txs_and_run_os(
+pub async fn execute_txs_and_run_os<T: Into<Transaction>>(
     state: CachedState<SharedState<DictStorage, PedersenHash>>,
     block_context: BlockContext,
-    txs: Vec<AccountTransaction>,
+    txs: Vec<T>,
```

Comment:
Don't make this generic, just pass it a `Vec<Transaction>` if possible. We will need to support a mix of account and L1 txs to prove real blocks.